.github/workflows/ci-e2e-tests.yml

```diff
@@ -92,6 +102,10 @@ jobs:
         chmod +x .github/scripts/e2e_test_bucket_creation.sh
         .github/scripts/e2e_test_bucket_creation.sh
 
+    - name: "Delay completion"
```

Comment:
There is also this one that makes it possible to extend or cut short the delay amount, if you think you may ever need it: https://github.com/scality/actions/blob/main/action-delay-job-completion/action.yaml

Reply:
I will use it